template/src/components/Hello.vue

```diff
@@ -21,12 +24,28 @@
 </template>
 
 <script>
+{{#store}}
+import { mapGetters, mapActions } from 'vuex'{{#if_eq lintConfig "airbnb"}};{{/if_eq}}
+{{/store}}
+
 export default {
   name: 'hello',
   data{{#unless_eq lintConfig "airbnb"}} {{/unless_eq}}() {
     return {
       msg: 'Welcome to Your Vue.js App'{{#if_eq lintConfig "airbnb"}},{{/if_eq}}
     }{{#if_eq lintConfig "airbnb"}};{{/if_eq}}
+  {{#store}}
+  },
+  computed: {
+    ...mapGetters([
+      'greeting'{{#if_eq lintConfig "airbnb"}},{{/if_eq}}
+    ]){{#if_eq lintConfig "airbnb"}},{{/if_eq}}
+  },
+  methods: {
+    ...mapActions([
+      'setGreeting'{{#if_eq lintConfig "airbnb"}},{{/if_eq}}
+    ]){{#if_eq lintConfig "airbnb"}},{{/if_eq}}
+  {{/store}}
```

Comment:
this `{{#store.../store}}` guard is a bit convoluted. It'll be better to simply have a single guard for the semi-colon, and another for actual content